motionplan/kinematic.go

```diff
@@ -36,6 +36,12 @@ func ComputePosition(model referenceframe.Frame, joints *pb.JointPositions) (spa
 // position of the end effector as a protobuf ArmPosition even when the arm is in an out of bounds state.
 // This is performed statelessly without changing any data.
 func ComputeOOBPosition(model referenceframe.Frame, joints *pb.JointPositions) (spatialmath.Pose, error) {
+	if joints == nil {
+		return nil, errors.New("joint positions should not be nil")
+	}
+	if model == nil {
+		return nil, errors.New("the model frame should not be nil")
```

Comment:
could the warnings be added here instead? It's a clearer place in the code where the problem of nil joint and model frame objects cause problems. In the client code, the warning just seems randomly placed.

Or warn in create-status, and how often does that error show up? I'd like something that allows you to progress writing your module while still telling you what's wrong.